data/snippets.ts

```diff
@@ -530,6 +530,12 @@ const coding = [
     keyword: "sql-select",
     type: "template",
   },
+  {
+    id: nanoid(),
+    name: "Get Name of Current Branch",
+    text: "git branch --contains | cut -d \" \" -f 2",
```

Comment:
Shouldn't we use `git branch --show-current`?

Reply:
If that's a better way, feel free to create a PR. We can change it.

Reply:
@aashutoshrathi 
Thanks. It’s better!